dotnet/samples/Concepts/Memory/VectorStore_VectorSearch_MultiStore_AzureAISearch.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Azure;
+using Azure.Identity;
+using Azure.Search.Documents.Indexes;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Connectors.AzureAISearch;
+using Microsoft.SemanticKernel.Connectors.AzureOpenAI;
+
+namespace Memory;
+
+/// <summary>
+/// An example showing how to use common code, that can work with any vector database, with an Azure AI Search instance.
+/// The common code is in the <see cref="VectorStore_VectorSearch_MultiStore_Common"/> class.
```

Comment:
The class name with _ looks odd to me. We use the _ in some of the classes that are tests for readability bit in this case it's a utility class.

Reply:
The naming is so that when someone looks at the folder, this is grouped with the test classes.
Previously I got feedback that people wanted to run samples for each db separately without it spinning up any infra for those that don't require it.  To make that work I have to split them into separate classes.